pandas/io/gbq.py

```diff
@@ -682,6 +692,15 @@ def read_gbq(query, project_id=None, index_col=None, col_order=None,
 
         .. versionadded:: 0.19.0
 
+    **kwargs: Arbitrary keyword arguments
+        configuration (dict): query config parameters for job processing.
+            For example:
```

Comment:
I think it will be helpful to have an example under the `Google BigQuery (Experimental)` heading in https://github.com/pandas-dev/pandas/blob/master/doc/source/io.rst?